### PR TITLE
Change approvers for department chiefs

### PR DESCRIPTION
**Pull request by sponkrashin**
Related to #230

### CODE REVIEW

File: server/Arcadia.Assistant.CSP/CspEmployeesInfoStorage.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ protected override async Task<LoadAllEmployees.Response> GetAllEmployees()
                             HireDate = x.HiringDate,
                             FireDate = x.FiringDate,
                             MobilePhone = x.MobilePhone,
-                            RoomNumber = x.RoomNumber,
+                            RoomNumber = x.RoomNumber != null ? x.RoomNumber.Trim() : null,
```
Comment (santee) [inline server/Arcadia.Assistant.CSP/CspEmployeesInfoStorage.cs]: I wonder how that happened... apparently someone needs to merge things back from master to develop :)

File: server/Arcadia.Assistant.Organization.Abstractions/OrganizationRequests/DepartmentsQuery.cs [hunk 2]
```diff
@@ -39,20 +40,29 @@ public DepartmentsQuery WithHead(string employeeId)
...
             return this.CloneTypewise();
         }
 
+        private DepartmentsQuery CloneTypewise()
+        {
+            var copy = new DepartmentsQuery
+            {
+                DepartmentId = this.DepartmentId,
+                AscendantDepartmentId = this.AscendantDepartmentId,
+                DepartmentHeadEmployeeId = this.DepartmentHeadEmployeeId
```
Comment (santee) [inline server/Arcadia.Assistant.Organization.Abstractions/OrganizationRequests/DepartmentsQuery.cs]: forgot your new field

File: server/Arcadia.Assistant.CSP/CspEmployeesInfoStorage.cs (hunk 1 above)
Comment (sponkrashin) [reply server/Arcadia.Assistant.CSP/CspEmployeesInfoStorage.cs]: Done. master -> develop -> current branch.

File: server/Arcadia.Assistant.Organization.Abstractions/OrganizationRequests/DepartmentsQuery.cs (hunk 2 above)
Comment (sponkrashin) [reply server/Arcadia.Assistant.Organization.Abstractions/OrganizationRequests/DepartmentsQuery.cs]: Fixed.

**Pull request merged**